docs/UserGuide.md

```diff
@@ -249,8 +249,8 @@ To achieve the best possible experience, avoid adding excessively long input to
 * The date of an appointment with your client
 * Should be in this format: `yyyy-mm-dd` 
   * e.g. `2023-05-15` 
-* Should be a valid date where:
-  * `yyyy` (year): should be a non-negative 4-digit number
+* Should be a valid date (cannot be a date in the past) where:
+  * `yyyy` (year): should be a positive 4-digit number 
```

Comment:
Is this enforced in the code? Will an error be thrown if a date in the past is detected?

Reply:
running the command 
```
aa 1 d/1999-10-10
```
works despite the date specified being in the past